src/parseTools.mjs

```diff
@@ -964,8 +964,7 @@ function from64Expr(x, assign = true) {
 }
 
 function toIndexType(x) {
-  if (MEMORY64 != 1) return x;
-  return `toIndexType(${x})`;
+  return to64(x);
```

Comment:
Should we replace calls from `toIndexType` to `to64`? Or do we think the distinction is important/might matter later?

Reply:
Yes, I thought of doing that.  Can always be done as followup I guess?

Reply:
Followup sounds fine, but also I am not sure if that is worthwhile. Unless we are certain we won't care about the difference later (which I am not sure of myself).